changelog/@unreleased/pr-6232.v2.yml

```diff
@@ -0,0 +1,7 @@
+type: fix
+fix:
+  description: |-
+    See PDS-293157.
+    Presently, as sweep progresses on each host, the lastSweptTimestamp metric is updated lazily per shard. This PR makes it so we update that metric periodically for all shards by reading from the persisted shard progress table.
```

Comment:
As you explicitly asked for feedback here:

This is fine! You can use backticks around things like lastSweptTimestamp e.t.c
If you really want to go all out - since this is a bug fix of sorts, what problem was caused by lazily updating per shard? This helps downstream users understand if there's anything they need to update / change their mental model.